index.html

```diff
@@ -141,6 +141,9 @@ <h3>File Sharing</h3>
                    be supported by servers as well as user agents.
    N15             It must be possible to support data exchange
                    in a worker.
+   N29             The application must be able to request user consent
+                   for one-way media and data only use cases in a
```

Comment:
This section is "file sharing", so "one-way media" should be broken out in a separate use case.

Reply:
By that, do you mean add the requirement to "Video Conferencing with a Central Server" as well?

I'm also wondering if it's clear enough what "user consent" is referring to. What are your opinions?